mesa/mesaPROTON_OC.py

```diff
@@ -778,7 +779,32 @@ def init_professional_links(self):
                 conn_pool = list(self.rng.choice(list(total_pool), conn, replace=False))
                 employee.makeProfessionalLinks(conn_pool)
 
+    def df_to_lists(self,df):
```

Comment:
Are you sure we need both df_to_dict and df_to_lists?

Reply:
I kept the data structures as in netlogo, also this function is very useful, especially when the transformed dataframes with this function are passed to the extra.pick_from_pair_list.